### ISSUE TITLE
peer review lab_02 by Miriam Ivaldi

### ISSUE THREAD

**Issue by theMiriam** [ludofiorio/labs_comp_inte#4]
Hi, 
I think you should have added more comments to make the code more clear, and also a read-me would have been useful.
I liked that you created 3 new strategies (eliminate_two_row, eliminate_one_row, leave_one_elem_row).
I think the basic idea of ​​your code is very good, but it seems that you used hill-climbig strategy rather than ES, you don't create a population and you don't have mutation and crossover functions. It's a good idea to calculate fitness by considering who starts the game for a good probability distribution. 
In conclusion, a suggestion to make the final results more understandable could be to add a print that shows the number of victories for the different strategies.

I hope this review was useful to you and good luck on your next labs.
